tests/integration/data/test-bundle-managed-etcd.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: integration-managed-etcd-test
```

Comment:
Let's use the default (i.e., `etcd`) as the default datastore for the whole suite, except `external-etcd` ofc. Maybe we can morph this one to dqlite?